modules/update-service-install-cli.adoc

```diff
@@ -0,0 +1,121 @@
+[id="update-service-install-cli_{context}"]
+= Installing the {product-title} Update Service by using the CLI
+
+You can use the {product-title} CLI to install the {product-title} Update Service Operator.
+
+.Procedure
+
+To install the {product-title} Update Service Operator by using the {product-title} CLI:
+
+. Create a Namespace for the {product-title} Update Service Operator:
+
+.. Create a Namespace object YAML file, for example, `update-service-namespace.yaml`, for the {product-title} Update Service Operator:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: FIXME-default-namespace
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true" <1>
+----
+<1> Set the `openshift.io/cluster-monitoring` label to enable operator recommended cluster monitoring on this namespace.
+
+.. Create the Namespace:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc create -f update-service-namespace.yaml
+----
+
+. Install the {product-title} Update Service Operator by creating the following objects:
+
+.. Create an Operator Group object YAML file, for example, `update-service-operator-group.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: update-service-operator-group
+spec:
+  targetNamespaces:
+  - FIXME-default-operand-namespace
+----
+
+.. Create an Operator Group object:
++
+[source,terminal]
+----
+$ oc -n FIXME-default-namespace create -f <file-name>.yaml
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc -n FIXME-default-namespace create -f update-service-operator-group.yaml
+----
+
+.. Create a Subscription object YAML file, for example, `update-service-subscription.yaml`:
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: update-service-subscription
+spec:
+  channel: "{product-version}" <1>
+  installPlanApproval: "Automatic"
+  source: "redhat-operators" <2>
+  sourceNamespace: "openshift-marketplace"
+  name: "FIXME-update-service-operator"
+----
+<1> Specify `{product-version}` as the channel.
+<2> Specify the name of the Operator channel. For clusters that do not use a custom Operator Lifecycle Manager (OLM), specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object created when you configured the Operator Lifecycle Manager (OLM).
```

Comment:
Shouldn't this
> Specify the name of the Operator channel.

be

`Specify the name of the CatalogSource that provides the Operator.`